beacon-chain/p2p/service.go

```diff
@@ -229,6 +232,13 @@ func (s *Service) Start() {
 	runutil.RunEvery(s.ctx, refreshRate, func() {
 		s.RefreshENR()
 	})
+	runutil.RunEvery(s.ctx, 1*time.Minute, func() {
```

Comment:
Will 1 minute be too verbose?

Reply:
With info logs, you will see it logged out every 5 blocks which is fine imo. Do you think it should be higher ? I can bump it up to 2 minutes in that case.

Reply:
Sounds good, we can always adjust it later